azure/packages/test/end-to-end-tests/src/test/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
-	"extends": ["../../../../../../common/build/build-common/tsconfig.test.json"],
-	"include": ["./**/*", "../*"],
+	"extends": "../../../../../../common/build/build-common/tsconfig.test.node16.json",
 	"compilerOptions": {
-		"rootDir": "../",
-		"outDir": "../../dist",
+		"rootDir": "./",
+		"outDir": "../../lib/test",
 		"types": ["mocha"],
 	},
+	"include": ["./**/*"],
```

Comment:
I'm guessing that at one time this project had *.ts files directly under './src'.  Currently, only /src/test/*.ts exists.

Reply:
I think there was a common pattern in the past that the include path had reference to the ../ ts files instead of a preferred reference to the product tsconfig project. And this was probably just copied from that pattern.